server/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -3066,6 +3072,11 @@ long getNumDocUpdates() {
         return numDocUpdates.count();
     }
 
+    @Override
+    public long getFlushTimeExcludingWaitingInMillis() {
```

Comment:
this is the total time spent on flush, maybe we should rename the field?

Reply:
Good point, I've changed it :+1: